Xamarin.Forms.ControlGallery.WindowsUniversal/MainPage.xaml.cs

```diff
@@ -127,6 +129,30 @@ void AddNativeControls(NestedNativeControlGalleryPage page)
 				}
 			);
 
+			{
+				var stackLayout = new StackLayout()
+				{
+					VerticalOptions = LayoutOptions.Start,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var label = new Label
+				{
+					Text = "Hello World!",
+					VerticalOptions = LayoutOptions.Center,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var bz56843TextBlock = new TextBlock
```

Comment:
But it doesn't document anything. It's just some random layout that makes the Windows version of this gallery page different from the other platforms. 

If you want to move it to another Page and document that "this is the (theoretical) repro for Bugzilla case 56843", I've got no objection to that; I just don't see the point of cluttering up the example page for this feature. 

I also wouldn't object to just removing it entirely, since we can't figure out how to make this bug reproduce in Control Gallery.